cfgov/v1/models/base.py

```diff
@@ -255,6 +297,10 @@ def get_context(self, request, *args, **kwargs):
         if self.schema_json:
             context['schema_json'] = self.schema_json
 
+        if self.search_description:
+            context['meta_description'] = self.search_description
+        else:
+            context['meta_description'] = self.get_meta_description()
         return context
```

Comment:
With search_description added to the function, we no longer need to check it here.

```suggestion
        context['meta_description'] = self.get_meta_description()
        return context
```